### ISSUE TITLE
Storage doesn’t allow for ChefSpec tests

### ISSUE THREAD

**Issue by jeffbyrnes** [evertrue/storage-cookbook#1]
Example failure:

```
  4) et_lids_api::default should include recipe et_logger::awslogs
     Failure/Error: let(:chef_run) { ChefSpec::Runner.new.converge(described_recipe) }
     Errno::ENOENT:
       No such file or directory @ rb_sysopen - /proc/mounts
     # /var/folders/lm/141jhsb55hq99sbxvfvn77zc0000gn/T/d20140908-88510-1eto7nz/cookbooks/storage/recipes/default.rb:17:in `readlines'
     # /var/folders/lm/141jhsb55hq99sbxvfvn77zc0000gn/T/d20140908-88510-1eto7nz/cookbooks/storage/recipes/default.rb:17:in `from_file'
     # /var/folders/lm/141jhsb55hq99sbxvfvn77zc0000gn/T/d20140908-88510-1eto7nz/cookbooks/et_rails_app/recipes/default.rb:17:in `from_file'
     # /var/folders/lm/141jhsb55hq99sbxvfvn77zc0000gn/T/d20140908-88510-1eto7nz/cookbooks/et_lids_api/recipes/default.rb:34:in `from_file'
     # ./spec/default_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./spec/default_spec.rb:49:in `block (3 levels) in <top (required)>'
```

**Comment by EdHurtig** [comment]
@jeffbyrnes Surprise surprise when I googled this issue it brought me right back to one of our own...

Anyways I can repro on the elk cookbooks. I will see if it is possible to swing this or at least return gracefully when running ChefSpec

```
 elk::default When all attributes are default, on an unspecified platform converges successfully
     Failure/Error: runner.converge(described_recipe)
     Errno::ENOENT:
       No such file or directory @ rb_sysopen - /proc/mounts
     # /var/folders/pk/40_7vjf94lg6377f4f3blwlh0000gn/T/d20150615-37872-fny7ds/cookbooks/storage/recipes/default.rb:22:in `readlines'
     # /var/folders/pk/40_7vjf94lg6377f4f3blwlh0000gn/T/d20150615-37872-fny7ds/cookbooks/storage/recipes/default.rb:22:in `from_file'
     # /var/folders/pk/40_7vjf94lg6377f4f3blwlh0000gn/T/d20150615-37872-fny7ds/cookbooks/elk_elasticsearch/recipes/install.rb:7:in `from_file'
     # /var/folders/pk/40_7vjf94lg6377f4f3blwlh0000gn/T/d20150615-37872-fny7ds/cookbooks/elk_elasticsearch/recipes/default.rb:9:in `from_file'
     # /var/folders/pk/40_7vjf94lg6377f4f3blwlh0000gn/T/d20150615-37872-fny7ds/cookbooks/elk/recipes/default.rb:19:in `from_file'
     # ./spec/unit/recipes/default_spec.rb:26:in `block (3 levels) in <top (required)>'
     # ./spec/unit/recipes/default_spec.rb:29:in `block (3 levels) in <top (required)>'
```

**Comment by EdHurtig** [comment]
Fixed by #5
